includes/polling/functions.inc.php

```diff
@@ -237,17 +237,14 @@ function poll_device($device, $options) {
         if ($options['m']) {
             foreach (explode(',', $options['m']) as $module) {
                 if (is_file('includes/polling/'.$module.'.inc.php')) {
-                    include 'includes/polling/'.$module.'.inc.php';
+                    load_poller_module($module, $device, $attribs);
                 }
             }
         }
         else {
             foreach ($config['poller_modules'] as $module => $module_status) {
                 if ($attribs['poll_'.$module] || ( $module_status && !isset($attribs['poll_'.$module]))) {
-                    $module_start = microtime(true);
-                    include 'includes/polling/'.$module.'.inc.php';
-                    $module_time = microtime(true) - $module_start;
-                    echo "Runtime for polling module '$module': $module_time\n";
+                    load_poller_module($module, $device, $attribs);
```

Comment:
This is an error from scrutinizer-ci.  The load_poller_module() only has two arguments, but three are given in these two calls.